website/docs/getting-started/for-engineers.md

```diff
@@ -140,3 +140,11 @@ Our styles, components and icons are supported by the following browsers:
 | Safari         | last 2 versions |
 | Firefox        | last 2 versions |
 | Microsoft Edge | last 2 versions |
+
+## Developer environment setup
```

Comment:
Might as well be 
```suggestion
## Code editor setup
```

Reply:
I prefer the second option ("Code editor setup") since the content is narrowed to this specific context